projects/counter/src/main.c

```diff
@@ -0,0 +1,47 @@
+#include "counter.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+#define COUNTER_TIMEOUT_MILI_SECS 500
```

Comment:
please just suffix the #define `_MS` to denote milli-seconds, to match with convention